### ISSUE TITLE
Native crash on M1

### ISSUE THREAD

**Issue by hrach** [arakelian/java-jq#21]
Hi, I'm trying the latest 1.3.0 with support for M1 and it doesn't work. Any idea what I'm doing wrong?

```
Jan 25, 2023 10:50:37 AM com.arakelian.jq.NativeLib getLocalCopy
INFO: Copying resource lib/darwin-x86_64/libjq.dylib to: /var/folders/ym/4hwqw5ss1f3f97hh0vv17nvm0000gn/T/1674640237923-0/libjq.dylib
Jan 25, 2023 10:50:37 AM com.arakelian.jq.NativeLib getNativeLibrary
INFO: jq library path: /var/folders/ym/4hwqw5ss1f3f97hh0vv17nvm0000gn/T/1674640237923-0
Assertion failed: (count <= len && "snprintf() output has been truncated"), function LOAD_ERROR, file dispatch.c, line 74.
```

```kotlin
        val request = ImmutableJqRequest.builder()
            .lib(ImmutableJqLibrary.of())
            .input("[]")
            .filter(".")
            .build()
        val filteredJson = request.execute().output
```

**Comment by arakelian** [comment]
Will take a look...

**Comment by MattyCi** [comment]
I'm also running into this issue.

**Comment by donarus** [comment]
need to update the JNA library to the 5.13.0, but the JNA-classes are pre-baked to the java-jq itself

**Comment by damianjacewicz-tomtom** [comment]
I have the same issue..

**Comment by arakelian** [comment]
I haven't been very active with this project for a couple of reasons, but mostly since I stopped using it myself because jq is not multithreaded, with no plans to support it. 

What I can do is make sure maven reflects latest merged code however, and I'll look at that shortly. Beyond that, I'll be looking for PRs to handle any fixes for apple silicon, which I use myself.

**Comment by fopinappb** [comment]
https://github.com/arakelian/java-jq/blob/9526168bbdc57a871d522ba5d6a09736d809e55b/src/main/java/com/arakelian/jq/NativeLib.java#L148-L153

I guess the lack of darwin arm64 library is the issue

**Comment by umutsahin** [comment]
Created PR including fix https://github.com/arakelian/java-jq/pull/25

@arakelian @hrach

**Comment by mahith2108** [comment]
Hi @arakelian ,  can you review the above PR (https://github.com/arakelian/java-jq/pull/25). @hrach Do let us know if there are any alternatives. @umutsahin If possible can you share the jar with M1 changes

**Comment by arakelian** [comment]
Should be fixed in version 2.0.0.
